src/blr/utils.py

```diff
@@ -38,3 +38,17 @@ def guess_paired_path(path: Path):
     if len(paths) == 1:
         return paths[0]
     return None
+
+
+def get_bamtag(pysam_read, tag):
+    """
+    Fetches tags from bam files. Return an empty value of the same type if not found.
+    :param pysam_read: pysam read object
+    :param tag: bam tag to fetch
+    :param type: what type to return if read does not have the tag
+    :return: bam tag value
+    """
+    try:
+        return pysam_read.get_tag(tag)
+    except KeyError:
+        return None
```

Comment:
Now `None` is returned by default by the `utils.get_tag` function upon `KeyError`. Do you think it would be useful to be able to specify a default?

Reply:
I used that from the beginning but found it better to return `None`. 

In cases where we have want to fetch a `tag_value` but there it is not defined, the original need of the `tag_val` should reasonably mean an absence of it should be handled differently.